schema/fixtures.js

```diff
@@ -292,6 +292,38 @@
               ]
             }
           ]
+        },
+        {
+          "id": 10,
+          "user": { "id": 5 },
+          "created_at": "2001-06-01T01:00:00",
+          "private_location": "1,2",
+          "project_ids": [ 1 ],
+          "private_geojson": { "type": "Point", "coordinates": [ "2", "1" ] },
+          "project_observations": [
+            {
+              "user_id": 1,
+              "uuid": "8cf0282d-c6f3-4947-9905-4937def371f4",
+              "project_id": 1,
+              "curator_coordinate_access": true
```

Comment:
`curator_coordinate_access ` isn't actually stored in the index (yet). Project observation preferences are fetched from the DB with `Observation.preloadProjectObservationPreferences` in `models/observation.js` . So if you wanted to be precise, you'd add a Preferences DB fixture with this data in it, and allow it to be fetched with all the other preloaded stuff in `preloadAllAssociations`.

This also happens to be part of the stuff that I'm working on adding to the index, but they won't be added exactly like this. Instead preferences will go in as an array of objects with name/value attributes (to control ES mapping), which I think will then be converted to an object in the observation constructor (to preserve backwards compatibility)